src/theme/palette.ts

```diff
@@ -8,10 +7,26 @@ interface OtherColorTypes {
   border: string
 }
 declare module "@mui/material/styles/createPalette" {
```

Comment:
I think you can also store this in `react-app-env.d.ts` if you wanted

Reply:
For the purpose of intuitive, I think it would be good to put on theme file.
Because button and typography is having `declaration` as well for inserting variants.